<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>1051</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>DB explorer</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>601</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutDBExplorer"/>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>490</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>410</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time exposition:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>430</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Magnetic fild:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>450</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>470</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance SD:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_timeExposition">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>410</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_magneticFild">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>430</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_distanceSD">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>470</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_temperature">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>450</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_wavelenght">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>490</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>490</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wavelenght</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>441</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutViewPlot"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>T-scan</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutPlotTscan2D"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>601</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutPlotTscan"/>
     </widget>
     <widget class="QListWidget" name="listWidgetTscan">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>396</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTscanAdd">
      <property name="geometry">
       <rect>
        <x>807</x>
        <y>396</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTscanClr">
      <property name="geometry">
       <rect>
        <x>807</x>
        <y>450</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clr</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTscanMinus">
      <property name="geometry">
       <rect>
        <x>807</x>
        <y>421</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTscanStart">
      <property name="geometry">
       <rect>
        <x>766</x>
        <y>493</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>H-scan</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Tab_4">
     <attribute name="title">
      <string>Phase diagram</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_Ks">
     <attribute name="title">
      <string>Ks</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutPlotKs2D"/>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>x</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="layoutPlotKsX"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>y</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="layoutPlotKsY"/>
       </widget>
      </widget>
     </widget>
     <widget class="QListWidget" name="listWidget_Ks">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>256</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonKs_add">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ks_minus">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ks_clr">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>410</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>470</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>interpolate:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxKsInterpolate">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>487</y>
        <width>47</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonKsInterpolate">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>487</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxKsCenter">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Center</string>
      </property>
      <widget class="QDoubleSpinBox" name="spinBoxKsCenterX">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKsCenterY">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKsAngle">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>angle:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonKsCenterOk">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>98</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonKsOk">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>490</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxKs">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Ks</string>
      </property>
      <widget class="QDoubleSpinBox" name="spinBoxKs1X">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKs1Y">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKs2X">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>67</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKs2Y">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>67</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>106</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>value:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinBoxKsValue">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>101</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>231</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Average</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonKsUpdateDB">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>490</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update DB</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Average</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionO_ptions"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionconnect"/>
   <addaction name="actiondisconnect"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionconnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionO_ptions">
   <property name="text">
    <string>O&amp;ptions</string>
   </property>
  </action>
  <action name="actiondisconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
